.gitignore

```diff
@@ -1,3 +1,17 @@
-*
```

Comment:
Would be nice if these * lines could remain. They hide the executables that are created by `nim c -r` etc..